src/dll/edbgEcmdDll.C

```diff
@@ -1556,9 +1557,9 @@ uint32_t dllGetMemProc(const ecmdChipTarget & i_target, uint64_t i_address, uint
       blockSize = 8;
     }
 
-    rc = adu_getmem_io(adu_target, i_address, buf, i_bytes, blockSize);
```

Comment:
There is no reason to have two separate calls to mem_read in the code.   The `true`/`false` at the end of the function calls could be made a variable that is established in the `i_mode == MEMPROC_CACHE_INHIBIT` check.  The the actual mem_read moved out of the conditional.

Reply:
Done